juriscraper/opinions/united_states/state/ill.py

```diff
@@ -84,6 +91,12 @@ def _get_docket_numbers(self) -> List[str]:
         """
         dockets_numbers = []
         for case in self.cases:
-            m = re.search(self.docket_re, case["neutral_citation"])
-            dockets_numbers.append(m.group("docket"))
+            match = re.search(self.docket_re, case["neutral_citation"])
+            if match:
+                dockets_numbers.append(match.group("docket"))
+            else:
+                logger.info(f"Could not find docket for case: '{case}'")
+                raise InsanityException(
+                    f"Could not find docket for case: '{case}'"
```

Comment:
See comment in illappct

Reply:
Same reasons explainded for **_illappct_**, but for method `_get_docket_numbers(self) -> List[str]`